.env.example

```
@@ -11,3 +11,14 @@ REDIS_CONTAINER_NAME=
 
+# OAuth state TTL
+STATE_TTL_SECONDS=
+
+# Backend base URL used for OAuth redirect_uri
+OAUTH_BASE_URL=
+
+# Kakao, Naver
+KAKAO_CLIENT_ID=
+KAKAO_CLIENT_SECRET=
+NAVER_CLIENT_ID=
+NAVER_CLIENT_SECRET=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)